lib/evmone/tracing.hpp

```diff
@@ -38,17 +39,19 @@ class Tracer
     }
 
     void notify_instruction_start(  // NOLINT(misc-no-recursion)
-        uint32_t pc, const ExecutionState& state) noexcept
+        uint32_t pc, intx::uint256* stack_top, int stack_height,
```

Comment:
The parameters are not beautiful here. The best option seems to be `std::span<const uint256>`, but we wait for C++20.

Reply:
In Silkworm we use `gsl::span` from https://github.com/microsoft/GSL.

Reply:
Any reason not using std::span from C++20?

Reply:
Because Silkworm is compatible with C++17.